wbstack/sync/01-clear.sh

```diff
@@ -3,4 +3,4 @@
 
 echo "Clearing git repo (except the files that we maintain here)"
 
-find ./ -mindepth 1 ! -regex '^./\(\.dockerignore|\.git\|\.github\|\.hadolint\.yml\|wbstack\|docker\-compose\.yml\|Dockerfile\|robots\.txt\|health\.php\|composer\.\w+\.json\|LocalSettings\.php\)\(/.*\)?' -delete
+find ./ -mindepth 1 ! -regex '^./\(\.dockerignore\|\.git\|\.github\|\.hadolint\.yml\|wbstack\|docker\-compose\.yml\|Dockerfile\|robots\.txt\|health\.php\|composer\.\w+\.json\|LocalSettings\.php\)\(/.*\)?' -delete
```

Comment:
Confused as to why GitHub is showing this change as I thought this had already been merged into `main` via #164...

Reply:
I think it's just how GH works; there is a commit with this change in it. The fact that after the "merge" with main it will be a No-Op isn't reflected here